tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -2690,6 +2871,59 @@ Status AddFusedContractionNode(
   return OkStatus();
 }
 
+Status FuseConv2DSwish(RemapperContext* ctx,
+                       const std::map<string, int>& matched_nodes_map,
+                       const std::set<int>& remove_node_indices,
+                       std::vector<bool>* invalidated_nodes,
+                       std::vector<bool>* nodes_to_delete) {
+  const GraphDef* graph = ctx->graph_view.graph();
```

Comment:
Could you please help remove the unused variable? Thank you!

Reply:
Done.